### PR TITLE
Added option to specify Sref, fixed units and minor edits

### PR DESCRIPTION
**Pull request by nwu63**
Addresses #131.

### CODE REVIEW

File: openaerostruct/functionals/total_aero_performance.py [hunk 1]
```diff
@@ -17,11 +17,13 @@ def initialize(self):
 
     def setup(self):
         surfaces = self.options['surfaces']
-
-        self.add_subsystem('sum_areas',
-             SumAreas(surfaces=surfaces),
-             promotes_inputs=['*S_ref'],
-             promotes_outputs=['S_ref_total'])
+        user_supplied_S_ref = False
```
Comment (johnjasa) [inline openaerostruct/functionals/total_aero_performance.py]: If this is set as False here, then directly used in the logic, it's always the case. Is that intended behavior?

File: openaerostruct/structures/create_rhs.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def setup(self):
         self.ny = surface['num_y']
 
         self.add_input('total_loads', val=np.zeros((self.ny, 6)), units='N')# dtype=data_type))
-        self.add_input('element_weights', val=np.ones((self.ny-1)), units='N')# dtype=data_type))
+        # self.add_input('element_weights', val=np.ones((self.ny-1)), units='N')# dtype=data_type))
```
Comment (johnjasa) [inline openaerostruct/structures/create_rhs.py]: Just delete this line here

File: openaerostruct/functionals/total_aero_performance.py (hunk 1 above)
Comment (nwu63) [reply openaerostruct/functionals/total_aero_performance.py]: Did you want this as an option which is passed in from the run script? I wasn't sure how to do that without cluttering up the whole thing.

Comment (johnjasa) [reply openaerostruct/functionals/total_aero_performance.py]: Right now the user would have to modify total_aero_performance.py to change what they're using? If that's the case, then yes, we do want it as an option.

**Comment by johnjasa** [comment]
Merging despite Travis timing out; we need to shorten the testing cycle

**Pull request merged**